pex/venv/pex.py

```diff
@@ -0,0 +1,466 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, print_function
+
+import errno
+import itertools
+import os
+import shutil
+from collections import defaultdict, Counter
+from textwrap import dedent
+
+from pex import pex_warnings
+from pex.common import safe_mkdir, pluralize, chmod_plus_x
+from pex.compatibility import is_valid_python_identifier
```

Comment:
This file was a move and splitup into functions from the `pex.tools.commands.venv` module.